.github/workflows/bigquery.yml

```
@@ -23,16 +26,31 @@ jobs:
         with:
           credentials_json: ${{ secrets.GOOGLE_CREDENTIALS }}
 
-      - uses: actions/setup-python@v4
+      # - uses: actions/setup-python@v4
+      #   with:
+      #     python-version: "3.x"
```

```suggestion
      #  reindex step uses conda
```
Given that the `reindex.py` imports conda, I think we can safely delete these commented lines.